src/Symfony/Component/Translation/Translatable.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation;
+
+use Symfony\Contracts\Translation\TranslatableInterface;
+
+/**
+ * @author Nate Wiebe <nate@northern.co>
+ */
+final class Translatable implements TranslatableInterface
```

Comment:
i'd still prefer to start without the interface, to see how it goes (whether this is needed in practice, rather then composition)

Reply:
This is probably fine, as I don't think this would be a BC break and wouldn't be much work to change. @apfelbox did voice concerns about this already, but they could probably still get around this by adding to the translator instead of the data object.